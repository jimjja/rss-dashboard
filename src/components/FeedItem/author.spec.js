import React from 'react';
import renderer from 'react-test-renderer';
import Author from './author';

describe('Author component', () => {
  it('should match snapshot', () => {
    const props = {
      author: 'Example author',
    };
    const tree = renderer.create(<Author {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
