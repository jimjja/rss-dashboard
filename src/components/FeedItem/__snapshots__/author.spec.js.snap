// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Author component should match snapshot 1`] = `
<h4
  style={
    Object {
      "display": "inline-block",
      "fontWeight": "normal",
      "margin": "0 .5em 0 0",
    }
  }
>
  Example author
</h4>
`;
